internal/service/module/errors.go

```diff
@@ -0,0 +1,5 @@
+package module
+
+import "errors"
+
+var ErrInvalidOption = errors.New("invalid option")
```

Comment:
With this, `errors.New("invalid option")` will be defined two times in this repo. Please consolidate them into `commonerrors`